docs_user/modules/con_bare-metal-provisioning-service-configurations.adoc

```diff
@@ -1,18 +1,16 @@
 [id="con_bare-metal-provisioning-service-configurations_{context}"]
 
-= Bare Metal Provisioning service configurations
+= {bare_metal} configurations
 
-The {bare_metal_first_ref} is configured by using configuration snippets. For more information about the configuration snippets, see xref:service-configurations_planning[Service configurations].
+You configure the {bare_metal_first_ref} by using configuration snippets. For more information about configuring the control plane with the {bare_metal}, see link:{customizing-rhoso}/index[{customizing-rhoso-t}].
```

Comment:
@hjensas By "configuration snippets"  do you mean "custom resources"?

Reply:
@hjensas Please see my question above. ^

Reply:
I thin "configuration snippets" is referring to the pieces (snippets) of configuration you can set in the `customServiceConfig` filed in the resource definition. For example:
```
spec:
  ...
  ironic:
    enabled: true
    template:
      ...
      ironicConductors:
      - customServiceConfig: |
          [neutron]
          cleaning_network = provisioning
          provisioning_network = provisioning
          rescuing_network = provisioning
```